crypto/x509v3/v3_purp.c

```diff
@@ -777,7 +811,11 @@ int X509_check_issued(X509 *issuer, X509 *subject)
         return X509_V_ERR_SUBJECT_ISSUER_MISMATCH;
 
     x509v3_cache_extensions(issuer);
+    if(issuer->ex_flags & EXFLAG_INVALID)
```

Comment:
Nit: miss these two nits in this version, sorry ;)

Reply:
Thanks a lot!